docs/apache-airflow-providers-google/operators/cloud/cloud_storage_transfer_service.rst

```diff
@@ -67,12 +67,12 @@ Using the operator
       :start-after: [START howto_operator_gcp_transfer_create_job_body_gcp]
       :end-before: [END howto_operator_gcp_transfer_create_job_body_gcp]
 
-.. exampleinclude:: /../../airflow/providers/google/cloud/example_dags/example_cloud_storage_transfer_service_aws.py
```

Comment:
I can see that you have fixed the documentation, but i think this file is still exists in example_dags folder. Could you please delete it?

Reply:
It is deleted for sure. I was confused as well.
The thing is that the github displays the system test file renamed (changed path), but not deleted and recreated. So the old file no longer exists.